pkg/ddl/partition.go

```diff
@@ -3487,26 +3497,18 @@ func (w *worker) onReorganizePartition(jobCtx *jobContext, job *model.Job) (ver
 			if !index.Unique {
```

Comment:
Same here, since global indexes currently must be global, we filter away non-unique indexes here. Should be removed when implementing non-unique global indexes.

Reply:
Let's keep the rest as-is, and remove them all when starting to support non-unique global indexes.